target/linux/lantiq/patches-5.10/0715-v5.17-net-lantiq_xrx200-convert-to-build_skb.patch

```diff
@@ -1,7 +1,7 @@
 From e015593573b3e3f74bd8a63c05fa92902194a354 Mon Sep 17 00:00:00 2001
 From: Aleksander Jan Bajkowski <olek2@wp.pl>
 Date: Tue, 4 Jan 2022 16:11:44 +0100
-Subject: [PATCH 715/715] net: lantiq_xrx200: convert to build_skb
+Subject: [PATCH] net: lantiq_xrx200: convert to build_skb
```

Comment:
Remove all this unrelated changes, thanks.

Reply:
My intention was to add the kernel version to the patch name.

Reply:
That's ok, but we don't need that subject change, do we?